org.openandroidinstaller.OpenAndroidInstaller.json

```diff
@@ -0,0 +1,166 @@
+{
+  "app-id": "org.openandroidinstaller.OpenAndroidInstaller",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "openandroidinstaller",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
```

Comment:
does it really support wayland?

Reply:
Yes, launching it without any X11 related permissions doesn't cause any errors and `xprop` is not able to see the window.